src/transforms/babel.js

```diff
@@ -312,7 +312,10 @@ async function getJSXConfig(asset, isSourceModule) {
  * Generates a babel config for stripping away Flow types.
  */
 function getFlowConfig(asset, isSourceModule) {
-  if (!isSourceModule && asset.contents.includes('@flow')) {
+  if (
+    !isSourceModule &&
+    asset.contents.substring(0, 20).indexOf('@flow') > -1
```

Comment:
https://jsperf.com/includes-vs-substring-includes-vs-indexof/1 Performance test to back up this choice